overlord/servicestate/service_control_test.go

```diff
@@ -179,9 +241,30 @@ func makeControlChange(c *C, st *state.State, inst *servicestate.Instruction, in
 	return chg
 }
 
+func (s *serviceControlSuite) TestControlDoesntCreateExecCommandTasks(c *C) {
```

Comment:
this test name should mention flags now? other test names need adjusting perhaps?